app/views/users/show.js.erb

```diff
@@ -1,26 +1,23 @@
 (function() {
```

Comment:
About this code, I think that we should have a js presenter and view instead of using UI logic inside the js.erb.